src/main/kotlin/no/nav/familie/tilbake/person/PersonService.kt

```diff
@@ -12,4 +12,16 @@ class PersonService(val pdlClient: PdlClient) {
         return pdlClient.hentPersoninfo(personIdent, fagsystem)
     }
 
+
+    fun hentAktørId(personIdent: String, fagsystem: Fagsystem): List<String> {
+        val hentIdenter = pdlClient.hentIdenter(personIdent, fagsystem)
+        return hentIdenter.data.pdlIdenter!!.identer.filter { it.gruppe == "AKTORID" && !it.historisk }.map { it.ident }
```

Comment:
hentIdenter.graphql burde ha historikk: false og hverken hentIdenter.graphql  eller IdentInformasjon burde inneholde feltet historisk. HVis du ikke trenger historiske forekomtser, så gir det ikke mening å hente dem for så å filtrere dem vekk.